### DISCUSSION TITLE
UI button to git commit the changes made to content *.md files?

### DISCUSSION THREAD

**Discussion by m-faith** [Zettlr/Zettlr#4144] [Ideas]
I'm looking for a GUI editor that I can give to non-coders so they can edit markdown files for their website (website made with a static site generator).

With automatic deployments (from git repo to webserver) all the person editing content needs to do is `git add . && git commit -m 'Content updates.' && git push`... but from a terminal? No, that's much too big of an ask. From a menu button in the Zettler UI??? YES!!!

Can this be done currently? If not... would this feature be considered for future enhancements?

**Comment by exanrauzer** [comment]
For now it doesn't. I think you can check hackmd or codimd - it may suits your needs.

**Reply by m-faith** [reply]
Whoops... this might have been better in the **Ideas** category instead of *Q&A*... do you think a feature like this would be appealing to others / in line with project goals?

**Comment by m-faith** [comment]
Would this feature be of interest?

**Reply by exanrauzer** [reply]
I can't tell, I do not code. However, version control or push/pull to GitHub sounds good - sure thing. Anyway, there is plenty of other aspects to cover and having Zettlr stable and steady on track as well as free of glitches is more worth resources.

**Reply by 87113** [reply]
I would like to have a git integration in Zettlr. :hand: :) But no presure ;)
